assignment0_bibek/MM_MV_Product.h

```diff
@@ -0,0 +1,9 @@
+#ifndef MM_MV_PRODUCT_H
+#define MM_MV_PRODUCT_H
+
+#include <vector>
+
+std::vector<std::vector<double>> M_M_Product(const std::vector<std::vector<double>>& matrixA, const std::vector<std::vector<double>>& matrixB);
```

Comment:
For function naming, I would suggest spelling out the words fully. Most people have wide screens and that avoids any confusion about what `M_M` or `M_V` means. Since C++ has operator overloading, I think having a function called `inner_product` that can take either two matrices or a matrix and a vector would be another good alternative.